src/Microsoft.Azure.SignalR.Management/RestHubLifetimeManager.cs

```diff
@@ -357,6 +359,76 @@ private static bool FilterExpectedResponse(HttpResponseMessage response, string
 
     public AsyncPageable<SignalRGroupConnection> ListConnectionsInGroup(string groupName, int? top = null, CancellationToken token = default)
     {
-        throw new NotImplementedException();
+        if (string.IsNullOrWhiteSpace(groupName))
+        {
+            throw new ArgumentException($"'{nameof(groupName)}' cannot be null or whitespace.", nameof(groupName));
+        }
+
+        if (top < 0)
+        {
+            throw new ArgumentOutOfRangeException(nameof(top), "The value must be greater than or equal to 0.");
+        }
+
+        return new PageableGroupMember(FetchPages, token);
+
+        async IAsyncEnumerable<Page<SignalRGroupConnection>> FetchPages(string? continuationToken, int? pageSizeHint)
+        {
+            // Calculate the api for the first page
+            var api = _restApiProvider.GetListConnectionsInGroupEndpoint(_appName, _hubName, groupName);
+            if (top.HasValue)
+            {
+                api.Query = new Dictionary<string, StringValues>
+                {
+                    ["top"] = top.Value.ToString(CultureInfo.InvariantCulture),
+                };
+            }
+            if (pageSizeHint.HasValue)
+            {
+                api.Query ??= new Dictionary<string, StringValues>();
+                api.Query["maxPageSize"] = pageSizeHint.Value.ToString(CultureInfo.InvariantCulture);
+            }
+            if (!string.IsNullOrEmpty(continuationToken))
+            {
+                api.Query ??= new Dictionary<string, StringValues>();
+                api.Query["continuationToken"] = continuationToken;
+            }
+            do
+            {
```

Comment:
we could also check if token.IsCancellationRequest then throw

Reply:
`HttpClient.SendAsync` has this check already. No need to check by ourselves.